cartography/intel/semgrep/deployment.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from typing import Any
+from typing import Dict
+
+import neo4j
+import requests
+
+from cartography.client.core.tx import load
+from cartography.models.semgrep.deployment import SemgrepDeploymentSchema
+from cartography.stats import get_stats_client
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+stat_handler = get_stats_client(__name__)
+_TIMEOUT = (60, 60)
```

Comment:
This timeout constant is duplicated a few times, should I move it to `__init__.py`? Or create some file like `common.py`?

Reply:
It's ok to keep this duplicated since it's not core to your plugin + it'll keep your PR scoped. A future PR can do that refactor.